Libraries/DirectMLX.h

```diff
@@ -82,7 +86,7 @@ inline UINT64 DMLCalcBufferTensorSize(
     UINT64 minimumImpliedSizeInBytes = 0;
     if (!strides)
     {
-        minimumImpliedSizeInBytes = sizes[0];
+        minimumImpliedSizeInBytes = sizes ? sizes[0] : 1;
```

Comment:
sizes is not an optional parameter to test nullability - it must be passed. If this is for the case of `dimensionCount = 0`, then I propose testing against `dimensionCount` or just saying:

```
        minimumImpliedSizeInBytes = 1;
        for (UINT i = 1; 0 < dimensionCount; ++i)
        {
            minimumImpliedSizeInBytes *= sizes[i];
        }
        minimumImpliedSizeInBytes *= elementSizeInBytes;
```

Reply:
Thanks for recommendation, using dimensionCount is better.